### PR TITLE
Add chinese and english documents of paddle.prod

### PR DESCRIPTION
**Pull request by gfwm2013**
180 76 141 178_documentation_docs_zh_api_cn_tensor_cn_prod_cn html (1)

### CODE REVIEW

**Review by zhhsplendid — APPROVED** [review]
LGTM

File: doc/fluid/api_cn/tensor_cn/prod_cn.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+.. py:function:: paddle.prod(x, axis=None, keepdim=False, dtype=None, name=None)
+
+
+
+对指定维度上的Tensor元素进行求乘积运算，并输出相应的计算结果。
+
+参数：
+    - **x** (Tensor) - 一个多维的Tensor，支持数据类型为float32，float64，int32，int64。
+    - **axis** (int，可选) - 求乘积运算的维度。如果是None，则计算所有元素的乘积并返回包含单个元素的Tensor，否则该参数必须在 :math:`[-x.ndim, x.ndim)` 范围内。如果 :math:`axis[i] < 0` ，则维度将变为 :math:`x.ndim + axis[i]` ，默认为None。
+    - **keepdim** (bool，可选) - 是否在输出 `Tensor` 中保留减小的维度。如 `keep_dim` 为True，否则结果张量的维度将比输入张量小，默认值为False。
```
Comment (TCChenlong) [inline doc/fluid/api_cn/tensor_cn/prod_cn.rst]: keepdim哈

**Review by TCChenlong — APPROVED** [review]
LGTM

**Review by wangchaochaohu — APPROVED** [review]
LGTM

File: doc/fluid/api_cn/tensor_cn/prod_cn.rst (hunk 1 above)
Comment (gfwm2013) [reply doc/fluid/api_cn/tensor_cn/prod_cn.rst]: Thanks，done.

**Pull request merged**